ArchUnitNET/Fluent/Syntax/Elements/ObjectsShould.cs

```diff
@@ -388,37 +387,30 @@ public TRuleTypeShouldConjunction BePrivateProtected()
 
         //Relation Conditions
 
-        public ShouldRelateToMethodMembersThat<TRuleTypeShouldConjunction, TRuleType> CallAnyMethodsThat()
```

Comment:
do we have an alternative?

Reply:
You can still just use `CallAnyMethods(...)`. I removed this condition because it leads to confusing results because the members of types which are just referenced and not in the imported assemblies are not stored in the architecture. Therefore the `CallAnyMethodsThat()` will not find any dependencies to these members.